poetry/config/config.py

```diff
@@ -45,12 +42,8 @@ def __init__(
         self._config = deepcopy(self.default_config)
         self._use_environment = use_environment
         self._base_dir = base_dir
-        self._config_source = DictConfigSource()
-        self._auth_config_source = DictConfigSource()
-
-    @property
```

Comment:
Funny this wasn't use -- was probably added defensively in case it was needed when the config code was written. Given it's dead, no harm in removing it.